linear-base.cabal

```diff
@@ -122,7 +124,7 @@ library
         containers,
         ghc-prim,
         hashable,
-        linear-generics,
+        linear-generics >= 0.2 && < 0.3,
```

Comment:
Is the upper bound necessary? Do you fear that something will go wrong with the next version of linear-generics?

Reply:
Just being conservative. Depending on future development of `LinearTypes` it's conceivable we could want to change the multiplicity polymorphic arrows to linear ones.

Reply:
Theorically, minor versions bumps before v1.0 can contain breaking changes, so I think that the upper bound make sense. However @treeowl at the moment we specify the version and hash of `linear-generics` in `stack.yaml`, so how does that work/interfere with `linear-base.cabal` version bounds?

Reply:
I don't use Stack in any meaningful way, so I don't have the slightest idea.